lib/cc/engine/bundler_audit/issue.rb

```diff
@@ -2,88 +2,6 @@ module CC
   module Engine
```

Comment:
This class is now empty, and the other two don't inherit from it or anything that I can see. Keeping it for a reason?